android/src/main/java/io/mosip/tuvali/wallet/Wallet.kt

```diff
@@ -14,19 +14,22 @@ import io.mosip.tuvali.cryptography.WalletCryptoBox
 import io.mosip.tuvali.cryptography.WalletCryptoBoxBuilder
 import com.facebook.react.bridge.Callback
 import io.mosip.tuvali.openid4vpble.Openid4vpBleModule
-import io.mosip.tuvali.retrymechanism.lib.BackOffStrategy
+import io.mosip.tuvali.common.retrymechanism.BackOffStrategy
 import io.mosip.tuvali.transfer.TransferReport
 import io.mosip.tuvali.transfer.Util
 import io.mosip.tuvali.verifier.GattService
 import io.mosip.tuvali.verifier.Verifier
 import io.mosip.tuvali.verifier.Verifier.Companion.DISCONNECT_STATUS
+import io.mosip.tuvali.wallet.exception.MTUNegotiationFailedException
 import io.mosip.tuvali.wallet.transfer.ITransferListener
 import io.mosip.tuvali.wallet.transfer.TransferHandler
 import io.mosip.tuvali.wallet.transfer.message.*
 import org.bouncycastle.util.encoders.Hex
 import java.security.SecureRandom
 import java.util.*
 
+private const val MTU_REQUEST_RETRY_DELAY_TIME = 250L
```

Comment:
(optional, nitpick) IMO, a unit like (milliseconds) would be a good addon for ourselves 6-10months down the line 
```suggestion
private const val MTU_REQUEST_RETRY_DELAY_TIME_MS = 250L
```